### ISSUE TITLE
Entware How to Uninstall

### ISSUE THREAD

**Issue by Salvora** [WDCommunity/wdpksrc#84]
**Platform**
My Cloud Mirror Gen2

**Application**
Entware

**Describe the bug**
I have installed the entware_20.11.17_MirrorG2.bin using GUI to my unit.
Firmware: 5.11.112
It was not working as expected though. So I tried to remove it.
But I cannot completely remove it.
Even after removing from gui, the installed packages have entware components and some startup and shutdown scripts still remain in the unit.
And I get confusing log outputs during reboots.
How do I completely remove it? Or
How can I restore my unit to stock?

**Comment by gabrielitos87** [comment]
Have you tried to check the folder /shares/Volume_1/Nas_Progs ?
Here are all the apps installed: you can remove the folder manually of the app you want to delete.

**Comment by Salvora** [comment]
Yeah but some startup scritps remain so it spams errors during reboots. Also, the app center is kind of different with entware.

**Comment by stefaang** [comment]
`/shares/Volume_1/Nas_Prog/entware` should be removed when you uninstall `entware`. 
The init.sh in that directory is the only one affecting boot.

`/shares/Volume_1/entware` contains all the entware data. Delete it to remove entware completely.
